server/channels/app/channel.go

```diff
@@ -542,9 +561,10 @@ func (a *App) createGroupChannel(c request.CTX, userIDs []string) (*model.Channe
 		Name:        model.GetGroupNameFromUserIds(userIDs),
 		DisplayName: model.GetGroupDisplayNameFromUsers(users, true),
 		Type:        model.ChannelTypeGroup,
+		Shared:      model.NewPointer(channelIsShared),
```

Comment:
if `channelID = ""`, `creator` is nil and we create a "shared channel", but down below we do not create a SharedChannel record and no invite being sent.